tfx/orchestration/pipeline.py

```diff
@@ -78,6 +78,7 @@ def __init__(self,
                metadata_connection_config: Optional[
                    metadata_store_pb2.ConnectionConfig] = None,
                components: Optional[List[base_node.BaseNode]] = None,
+               sort_components: Optional[bool] = True,
```

Comment:
Can we do this without changing the pipeline code?  For example, a function in your code that takes a Pipeline object and returns a list of topologically sorted components?  We shouldn't require that the user construct their pipeline with this argument to have this work with the K8s dag runner.

Reply:
Yes, I reverted this change and I will look for a different way to implement this in the separate PR.